src/frontend/src/btc/components/convert/BtcConvertForm.svelte

```diff
@@ -58,7 +58,7 @@
 		{:else if nonNullish($hasPendingTransactionsStore)}
 			<div class="mb-4" data-tid="btc-convert-form-send-warnings">
 				<BtcSendWarnings
-					utxosFee={$storeUtxosFeeData?.utxosFee}
+					utxosFee={nonNullish(sendAmount) ? $storeUtxosFeeData?.utxosFee : undefined}
```

Comment:
Maybe create a local state to derive the information? This would make the logic a bit more readable. WDYT?

Do we need a test or it's kind of covered?

Reply:
Sounds good, updated.

Re. tests - correct, we already have a test case that checks if `BtcSendWarnings` appears correctly on `nonNullish($hasPendingTransactionsStore)`.